tools/ardusub.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python
+
+from pymavlink import mavutil
+
+master = mavutil.mavlink_connection('udpout:localhost:9000')
+
+while True:
```

Comment:
You need a time.sleep here or if the pixhawk is not connected the cpu will run at 100%.